src/plots/cartesian/helpers.js

```diff
@@ -0,0 +1,57 @@
+/**
+* Copyright 2012-2020, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+function getAxId(ax) {
+    return ax._id;
+}
```

Comment:
I know you just moved this from one place to another, but to my mind it's harder to remember that this function exists somewhere than to just remember that axes have an `_id` attribute. Non-blocking, but I'd 🔪

Reply:
Done in https://github.com/plotly/plotly.js/pull/4775/commits/a6662067ae8bb6b62f124f5cd0ed1272e1fcc00b.